obspy/clients/fdsn/tests/test_mass_downloader.py

```diff
@@ -20,10 +20,7 @@
 
 import pytest
 
-if sys.version_info.major == 2:
-    from httplib import HTTPException
-else:
-    from http.client import HTTPException
+from http.client import HTTPException
```

Comment:
This should also move to the standard library block above `pytest`.

Reply:
done :+1: